test/WorkerChannelTests.ts

```diff
@@ -455,6 +455,39 @@ describe('WorkerChannel', () => {
     assertInvocationSuccess(expectedOutput, expectedReturnValue);
   });
 
+    it ('returns and serializes falsy values', () => {

```

Comment:
nit on spacing :) also sorry would you mind breaking it out into separate steps? like `returns and serializes falsy value ""`

Reply:
ah, I didn't catch the spacing :)
And sure, will do!